app/lib/MyAccountsStore.js

```diff
@@ -83,6 +83,25 @@ export default class MyAccountsStore {
         : this.addAccount(konnector, account.values)
     result
     .then(account => {
+      return cozy.client.fetchJSON('PATCH', konnector.links.permissions, {
+        data: {
+          id: konnector.links.permissions,
+          type: 'io.cozy.permissions',
+          attributes: {
+            type: 'app',
+            source_id: konnector._id,
+            permissions: {
+              saveFolder: {
+                type: 'io.cozy.files',
+                values: [folder._id]
+              }
+            }
+          }
+        }
+      })
+      .then(() => account)
```

Comment:
The "account" variable coming from connectAccount does not have an _id attribute, the account is not created yet at that moment. That is why I have to use the account returned from addAccount and pass it through the promise chain. I could also use a more global variable but I do not think it is nicer.